sympy/physics/fluid/fluid_prop.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
```

Comment:
NumPy is used nowhere in the file. Also not everyone has numpy installed. Please remove this.